commons-math-legacy/src/main/java/org/apache/commons/math4/legacy/analysis/function/Logit.java

```diff
@@ -182,8 +182,8 @@ public DerivativeStructure value(final DerivativeStructure t)
             //  for x close to hi the signs will always be +inf)
             // this is probably overkill, since the call to compose at the end
             // of the method will transform most infinities into NaN ...
-            for (int i = 2; i < f.length; ++i) {
```

Comment:
This is wrong. f[0 and f[1] have been set and should be duplicated through the array.